app/helpers/application_helper.rb

```diff
@@ -334,7 +334,7 @@ def admin_links_for(community)
       :topic => :general,
       :text => t("admin.left_hand_navigation.preview"),
       :icon_class => icon_class("eye"),
-      :path => search_path(big_cover_photo: true),
+      :path => homepage_without_locale_path(big_cover_photo: true),
```

Comment:
In other places where this path helper is used, we also set `locale: nil`. Which makes me think, should we set it also here?